apps/examples/wifi_manager/wm_test/wm_test_dtls_client.c

```diff
@@ -0,0 +1,292 @@
+/****************************************************************************
+ *
+ * Copyright 2018 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License\n");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+#include <sys/types.h>
+#include <stdio.h>
+#include <string.h>
+#include "wm_test_log.h"
+#include "mbedtls/config.h"
+#include "mbedtls/net.h"
+#include "mbedtls/debug.h"
+#include "mbedtls/ssl.h"
+#include "mbedtls/entropy.h"
+#include "mbedtls/ctr_drbg.h"
+#include "mbedtls/error.h"
+#include "mbedtls/certs.h"
+#include "mbedtls/timing.h"
+
+#define SERVER_PORT "4433"
+#define READ_TIMEOUT_MS 1000
+#define MAX_RETRY 5
+#define DEBUG_LEVEL 0
+#define WT_BUF_SIZE 1024
+#define TAG "[WTDC]"
+
+static unsigned char buf[WT_BUF_SIZE];
+
+static void my_debug(void *ctx, int level, const char *file, int line, const char *str)
+{
+	((void)ctx);
+	((void)level);
+	printf("%s:%04d: %s", file, line, str);
+	fflush((FILE *)ctx);
+}
+
+int wm_test_dtls_client(char *server_addr, int msg_size, char *mac_buf, char *recv_mac_buf)
+{
+	int ret = 0;
+	uint32_t flags;
+	const char *pers = "wm_test_dtls_client";
+	int retry_left = MAX_RETRY;
+
+	mbedtls_net_context server_fd;
+	mbedtls_entropy_context entropy;
+	mbedtls_ctr_drbg_context ctr_drbg;
+	mbedtls_ssl_context ssl;
+	mbedtls_ssl_config conf;
+	mbedtls_x509_crt cacert;
+	mbedtls_timing_delay_context timer;
+#if defined(MBEDTLS_DEBUG_C)
+	mbedtls_debug_set_threshold(DEBUG_LEVEL);
+#endif
+
+	/* 1. Initialize the RNG and the session data */
+	mbedtls_net_init(&server_fd);
+	mbedtls_ssl_init(&ssl);
+	mbedtls_ssl_config_init(&conf);
+	mbedtls_x509_crt_init(&cacert);
+	mbedtls_ctr_drbg_init(&ctr_drbg);
+
+	WT_LOG(TAG, "Seeding the random number generator...");
+	fflush(stdout);
```

Comment:
is fflush(stdout) required with WT_LOG also

Reply:
It is required.